appinventor/components-ios/src/ViewComponent.swift

```diff
@@ -25,6 +25,48 @@ import Foundation
   fileprivate var _row = kDefaultRowColumn
 
   private var _visible = true
+  
+  
+  private var _left: Int = -1 //ComponentConstants.DEFAULT_X_Y  // Default invalid position
+  private var _top: Int = -1   // Default invalid position
+      
+  // Getter methods that RelativeLayout will use to query position
+  open func left() -> Int {
+      return _left
+  }
+  
+  open func top() -> Int {
+      return _top
+  }
```

Comment:
Given that the var getters for Left and Top do the exact same thing, these functions seem like they could be removed.